docs/functions/internal.rst

```diff
@@ -54,6 +54,26 @@ Formatting
 
 .. _`ANSI escape color code`: https://en.wikipedia.org/wiki/ANSI_escape_code#Colors
 
+Battery
+---
+
+.. function:: __lp_battery_acpi -> var:lp_battery, var:lp_battery_status
+
+   Returns the status and remaining capacity of the battery, as reported by `acpi`
+   tool. This requires `acpi` to be installed.
+
+   If set, uses :attr:`LP_BATTERY_ID` to specify which battery to monitor.
+
+   .. versionadded:: 2.1
+
+.. function:: __lp_battery_sysfs -> var:lp_battery, var:lp_battery_status
+
+   Returns the status and remaining capacity of the battery, using `sysfs`.
+   This is the default method.
```

Comment:
This should mention that it returns the first battery found if there are multiple.